packages/react-native-editor/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ For each user feature we should also add a importance categorization label  to i
 
 ## Unreleased
 -   [*] [internal] Move InserterButton from components package to block-editor package [#56494]
+-   [**] Editor displays network status when offline [#56627]
```

Comment:
If we were to merge this PR within the next week with this work behind a `__DEV__` flag, this may not be the most accurate release note as it would not display network status until connected to the connectivity hook. Do we have a precedent for release notes with dev-only changes? Perhaps just `[internal]`?